packages/urql/updates/index.js

```diff
@@ -1,90 +1,129 @@
       if (result[info.fieldName].success) {
         cache.invalidate({ __typename: 'ModerationAction', id: args.moderationActionId })
       }
     },
-    createComment,
+
+    createComment: (result, args, cache, info) => {
+      const parentId = args?.data?.parentCommentId || args?.data?.postId
+      const parentType = args?.data?.parentCommentId ? 'Comment' : 'Post'
+      const fieldName = args?.data?.parentCommentId ? 'childComments' : 'comments'
+      makeAppendToPaginatedSetResolver({
+        parentType,
+        parentId,
+        fieldName
+      })(result, args, cache, info)
+    },
+
+    createMessage: makeAppendToPaginatedSetResolver({
+      parentType: 'MessageThread',
+      fieldName: 'messages',
+      parentIdGetter: get('data.messageThreadId')
+    }),
+
     createModerationAction: (result, args, cache, info) => {
       if (result[info.fieldName].id) {
         const postId = args?.data?.postId
         cache.invalidate({ __typename: 'Post', id: postId })
       }
     },
+
     deleteComment: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
         cache.invalidate({ __typename: 'Comment', id: args.id })
       }
     },
+
     deletePost: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
         cache.invalidate({ __typename: 'Post', id: args.id })
       }
     },
+
     joinProject: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
         cache.invalidate(cache.keyOfEntity({ __typename: 'Post', id: args.id }), 'members')
       }
     },
+
     leaveProject: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
         cache.invalidate(cache.keyOfEntity({ __typename: 'Post', id: args.id }), 'members')
       }
     },
+
     login: (result, args, cache, info) => {
       if (!result?.error) {
         cache.updateQuery({ query: meCheckAuthQuery }, data => result?.login)
       }
     },
+
     logout: (result, args, cache, info) => {
       if (result?.logout?.success) {
         cache.updateQuery({ query: meCheckAuthQuery }, data => ({ me: null }))
       }
     },
+
     recordClickthrough: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
         const postId = args?.postId
         cache.invalidate({ __typename: 'Post', id: postId })
       }
     },
+
     removePost: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
         cache.invalidate({ __typename: 'Post', id: args.postId })
       }
     },
+
     removeProposalVote: (result, args, cache, info) => { 
       cache.invalidate({ __typename: 'Post', id: args.postId })
     },
+
     pinPost: (result, args, cache, info) => {
       if (result[info.fieldName].success) {
-        // Any Post invalidation will result in the full Group/Stream query being re-fetched.
+        // Note: Any Post invalidation will result in the full Group/Stream query being re-fetched.
```

Comment:
Acknowledged

Reply:
Sorry if I was repetitive :) Just putting these in there for later tuning/refactoring. As it is the perf on this stuff is not a big deal.

Reply:
no no, just wanted to note I read this! I just wanted an emoji react really